docs/ui/index.rst

```diff
@@ -0,0 +1,133 @@
+.. _ref_ui_overview:
+
+==
+UI
+==
+
+EdgeDB UI is a beautiful, feature-rich admin panel baked directly into all 
```

Comment:
To avoid making it seem as though we're branding the UI "EdgeDB UI." We refer to it more generically otherwise, so this is consistent with that.
```suggestion
EdgeDB's UI is a beautiful, feature-rich admin panel baked directly into all 
```